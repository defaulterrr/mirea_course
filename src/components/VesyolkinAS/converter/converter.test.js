import React from 'react';
import { create } from 'react-test-renderer';

import Converter from './converter';

describe('[RENDER] Converter - Snapshot test', () => {
  test('Matches the snapshot', () => {
    const captcha = create(<Converter />);
    expect(captcha.toJSON()).toMatchSnapshot();
  });
});
