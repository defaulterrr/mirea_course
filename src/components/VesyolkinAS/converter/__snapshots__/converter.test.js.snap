// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[RENDER] Captcha - Snapshot test Matches the snapshot 1`] = `
<div
  style={
    Object {
      "background": "#73716e",
    }
  }
>
  <h3>
    Конвертер валют курс на 21.05.2020
  </h3>
  <p>
    <select
      name="type1"
      onChange={[Function]}
      value=""
    >
      <option
        selected="selected"
      >
        Исходная валюта
      </option>
      <option
        value="1"
      >
        $
      </option>
      <option
        value="1.1"
      >
        €
      </option>
      <option
        value="0.014"
      >
        ₽
      </option>
      <option
        value="0.0093"
      >
        ￥
      </option>
    </select>
    <input
      name="num1"
      onInput={[Function]}
      placeholder="Число"
      type="number"
      value=""
    />
    <select
      name="type2"
      onChange={[Function]}
      value=""
    >
      <option
        selected="selected"
      >
        Конвертированая валюта
      </option>
      <option
        value="1"
      >
        $
      </option>
      <option
        value="1.1"
      >
        €
      </option>
      <option
        value="0.014"
      >
        ₽
      </option>
      <option
        value="0.0093"
      >
        ￥
      </option>
    </select>
    <p>
      0
    </p>
  </p>
</div>
`;

exports[`[RENDER] Converter - Snapshot test Matches the snapshot 1`] = `
<div
  style={
    Object {
      "background": "#73716e",
    }
  }
>
  <h3>
    Конвертер валют курс на 21.05.2020
  </h3>
  <p>
    <select
      name="type1"
      onChange={[Function]}
      value=""
    >
      <option
        selected="selected"
      >
        Исходная валюта
      </option>
      <option
        value="1"
      >
        $
      </option>
      <option
        value="1.1"
      >
        €
      </option>
      <option
        value="0.014"
      >
        ₽
      </option>
      <option
        value="0.0093"
      >
        ￥
      </option>
    </select>
    <input
      name="num1"
      onInput={[Function]}
      placeholder="Число"
      type="number"
      value=""
    />
    <select
      name="type2"
      onChange={[Function]}
      value=""
    >
      <option
        selected="selected"
      >
        Конвертированая валюта
      </option>
      <option
        value="1"
      >
        $
      </option>
      <option
        value="1.1"
      >
        €
      </option>
      <option
        value="0.014"
      >
        ₽
      </option>
      <option
        value="0.0093"
      >
        ￥
      </option>
    </select>
    <p>
      0
    </p>
  </p>
</div>
`;
